### ISSUE TITLE
nx-remotecache-azure fails on Node 12+ with Cannot find module 'fs/promises

### ISSUE THREAD

**Issue by amojicamu** [NiklasPor/nx-remotecache-custom#1]
Hi,

While trying the nx-remotecache-azure library, it failed with 'Cannot find module 'fs/promises'. I traced the issue to this repo. I'll submit a PR to fix it.

Thanks in advance

**Comment by NiklasPor** [comment]
Hey @amojicamu – are you sure that your version is Node 12+?

Seems like the promise API is available starting at node 12:

![image](https://user-images.githubusercontent.com/13211347/133134383-beece06e-4306-417d-afc4-49aa9bdfc7c4.png)

https://nodejs.org/api/fs.html#fs_promises_api

**Comment by NiklasPor** [comment]
And if so, could you please try again with node 12+ – if this resolves the error we could skip the PR. (And I'd suggest you to use some actively maintained node version.)

![image](https://user-images.githubusercontent.com/13211347/133135195-698be8f1-8662-4517-a263-a32cfbaa7551.png)

**Comment by amojicamu** [comment]
Hi, and welcome back. 

Here is what I have: `node --version: v12.22.1`

It seems there is some difference between the way it's exposed in Node v12 and v14.

Here's were I got the information.
https://stackoverflow.com/questions/64725249/fs-promises-api-in-typescript-not-compiling-in-javascript-correctly

Do you think it would be possible to maintain the include like this so there is more time to migrate? That is if this flavor of include also works for v14 of course

**Comment by amojicamu** [comment]
Hello Again,

I have confirmed that the modified include works in Node v14 as well as v12. I have v14 in my personal box.

`node --version: v14.12.0`

```
> nx run aztest:test
 PASS   aztest  apps/aztest/src/app/app.component.spec.ts
  AppComponent
    √ should create the app (84 ms)
    √ should have as title 'aztest' (14 ms)
    √ should render title (14 ms)

Test Suites: 1 passed, 1 total
Tests:       3 passed, 3 total
Snapshots:   0 total
Time:        4.086 s
Ran all test suites.
------------------------------------------------------------------------------
Stored output to remote cache: Azure Blob Storage
Hash: b90d49e2921237617784bbf0263b3f05c62e414fe975a4d81292f55798be5aae
------------------------------------------------------------------------------

———————————————————————————————————————————————

>  NX   SUCCESS  Running target "test" succeeded
```

**Comment by msamtleben** [comment]
Hey, I also have the same problem.
As @NiklasPor  already showed, the `fs/promises` import does only work in node v14+.
This lib depends on this import though which makes it incompatible with node v12.
@amojicamu 's pr fixes it though. Please consider merging it soon.

**Comment by NiklasPor** [comment]
The PR will be merged this evening. This and the `nx-remotecache-azure` package will receive a patch release.

**Comment by NiklasPor** [comment]
Fixed in PR #2.

Releases:
- nx-remotecache-azure@0.0.8
- nx-remotecache-custom@0.0.5

Thanks for your contribution @amojicamu 🚀
